test/modules/overlays.js

```diff
@@ -263,23 +263,23 @@
             var zoom = viewport.viewportToImageZoom(viewport.getZoom(true));
             var expectedWidth = 124 * zoom;
             var expectedHeight = 132 * zoom;
-            Util.assessNumericValue(assert, $("#overlay").width(), expectedWidth, epsilon,
+            Util.assessNumericValue(assert, $("#overlay-wrapper-overlay").width(), expectedWidth, epsilon,
                 "Width mismatch " + contextMessage);
-            Util.assessNumericValue(assert, $("#overlay").height(), expectedHeight, epsilon,
+            Util.assessNumericValue(assert, $("#overlay-wrapper-overlay").height(), expectedHeight, epsilon,
                 "Height mismatch " + contextMessage);
 
 
             expPosition = viewport.imageToViewerElementCoordinates(
                 new OpenSeadragon.Point(400, 500));
-            actPosition = $("#fixed-overlay").position();
+            actPosition = $("#overlay-wrapper-fixed-overlay").position();
```

Comment:
Same here, it seems like we should be checking the position of the actual overlay, not the wrapper.

Reply:
I think position needs to be controlled by the wrapper otherwise the flipping of the content doesn't work... not only that but overlays without scaling get positioned wrong.

Reply:
I agree that the wrapper controls the positioning. What I'm saying is, don't we want to verify that the overlay is actually positioned correct (by the wrapper)? It seems like directly testing the overlay's position would be better.

Reply:
That sounds good. @BeebBenjamin what happens if we remove the wrapper from all of the position tests?

Reply:
All the position tests fail.  Trying to understand why the outer (wrapper) position is correct but not the overlay...

Reply:
Examples:
```bash
Message: Not scaled overlay position.y should adjust to rotation. Expected:0 Found: 250.0000000000001 Variance: 1
Not scaled overlay position.x should adjust to rotation. Expected:0 Found: 400 Variance: 1
```

Reply:
Forgive me if I am wrong, but could it be the way we get the position?

https://api.jquery.com/position/

Reply:
Doesn't this get the position of the overlay relative to it's parent i.e. the wrapper?

Reply:
Oh yeah, good point! Sorry for the wild goose chase on that. 

We could probably write a helper function to get the actual proper location, perhaps using [getBoundingClientRect](https://developer.mozilla.org/en-US/docs/Web/API/Element/getBoundingClientRect). That said, I'm also okay with going back to testing the wrapper if you'd rather leave it at that.

Reply:
It's OK, I managed to tear out the use of JQuery where it fails to get position.